test/albums/queries.spec.js

```diff
@@ -0,0 +1,64 @@
+const nock = require('nock');
+
+const { query } = require('../server.spec');
+const { getAlbum, getAlbums } = require('./graphql');
+const albumMocks = require('../utils/mocks/albums');
+
+describe('albums', () => {
+  describe('queries', () => {
+    it('should retrieve an album', () => {
+      const id = 1;
+      albumMocks.mockGetAlbumOK(id);
+      albumMocks.mockGetPhotosOK(id);
+      return query(getAlbum(id)).then(response => {
+        expect(response.data).toHaveProperty('album', expect.any(Object));
+        const { album } = response.data;
+        expect(album).toHaveProperty('title', expect.any(String));
```

Comment:
why would you expect `any` string if you should know exactly the value? Or are you generating them randomly?

Reply:
Nop, let's say it was something that went over my head, because I even designed the mocks to be able to send the desired response. I'll change it